Bomb.py

```diff
@@ -0,0 +1,40 @@
+import pygame
+from constants import *
+
+class Bomb:
+    def __init__(self, x, y, power):
+        self.x = x
+        self.y = y
+        self.power = power
+        self.timer = 0
+
+    def __del__(self):
+        pass
+
+    def plant(self, win):
+        file = 0
+        if self.timer <= FIRST_BOMB_STATE:
+            file = BOMB_IMAGE_1
+        elif self.timer <= SECOND_BOMB_STATE:
+            file = BOMB_IMAGE_2
+        elif self.timer <= THIRD_BOMB_STATE:
+            file = BOMB_IMAGE_3
+        elif self.timer <= FOURTH_BOMB_STATE:
+            file = FIRE_IMAGE
+        else:
+            return
```

Comment:
Im not sure about `plant` name, since you actually draw the bomb here, not plant it. `draw` would make sense, as we do it in player and map aswell